### PR TITLE
update 6.0 release notes dm compatibility

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Add a missing note.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/8917
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: releases/release-6.0.0-dmr.md [hunk 1]
```diff
@@ -338,6 +338,8 @@ TiDB v6.0.0 is a DMR, and its version is 6.0.0-DMR.
     - Because of internal mechanism changes, the interface related to task management cannot be compatible with the previous experimental version. You need to refer to the new [DM OpenAPI documentation](/dm/dm-open-api.md) for adaptation.
 - DM changes the methods to resolve conflicts during the full import phase
     - A `loader.<name>.on-duplicate` parameter is added. The default value is `replace`, which means using the new data to replace the existing data. If you want to keep the previous behavior, you can set the value to `error`. This parameter only controls the behavior during the full import phase.
+- DM must use with the corresponding `dmctl` CLI
```
Comment (sunzhaoyang) [inline releases/release-6.0.0-dmr.md:341]:
```suggestion
- DM must use with the corresponding version of `dmctl`
```

**Review by sunzhaoyang — APPROVED** [review]
rest LGTM

File: releases/release-6.0.0-dmr.md [hunk 2]
```diff
@@ -338,6 +338,8 @@ TiDB v6.0.0 is a DMR, and its version is 6.0.0-DMR.
...
+    - Because internal mechanism changes, after you upgrade DM to v6.0.0, you must upgrade `dmctl` to the same version.
```
Comment (shichun-0415) [inline releases/release-6.0.0-dmr.md:342]:
```suggestion
- To use DM, you should use the corresponding `dmctl` CLI
    - Due to internal mechanism changes, after upgrading the DM to v6.0.0, you should also upgrade `dmctl` to v6.0.0.
```
Does the same version indicate v6.0.0?

File: releases/release-6.0.0-dmr.md [hunk 3]
```diff
@@ -338,6 +338,8 @@ TiDB v6.0.0 is a DMR, and its version is 6.0.0-DMR.
     - Because of internal mechanism changes, the interface related to task management cannot be compatible with the previous experimental version. You need to refer to the new [DM OpenAPI documentation](/dm/dm-open-api.md) for adaptation.
 - DM changes the methods to resolve conflicts during the full import phase
     - A `loader.<name>.on-duplicate` parameter is added. The default value is `replace`, which means using the new data to replace the existing data. If you want to keep the previous behavior, you can set the value to `error`. This parameter only controls the behavior during the full import phase.
+- DM must use with the corresponding version of `dmctl`
+    - Because internal mechanism changes, after you upgrade DM to v6.0.0, you must upgrade `dmctl` to the same version.
```
Comment (ran-huang) [inline releases/release-6.0.0-dmr.md:342]:
```suggestion
- To use DM, you should use the corresponding version of `dmctl`
    - Due to internal mechanism changes, after upgrading DM to v6.0.0, you should also upgrade `dmctl` to v6.0.0.
```

**Comment by ran-huang** [comment]
@shichun-0415 PTAL again

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**